clientv2/linux/data_change_api.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
```

Comment:
maybe some code is just copied from the clientv1, but i would put current year to the licence header, everywhere